phoenix5-spark/src/it/java/org/apache/phoenix/spark/OrderByIT.java

```diff
@@ -46,15 +46,9 @@
 import org.apache.spark.sql.sources.v2.DataSourceOptions;
 import org.junit.Test;
 import org.junit.experimental.categories.Category;
-import org.junit.Ignore;
-
-
-
-import scala.Option;
-import scala.collection.JavaConverters;
 
 @Category(ParallelStatsDisabledTest.class)
-public class OrderByIT extends BaseOrderByIT {
```

Comment:
Bad extends, BaseOrderByIT runs Phoenix based IT tests. These tests are out of context here.